ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/FakeKafkaServiceTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.tools;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import io.confluent.ksql.test.serde.string.StringSerdeSupplier;
+import java.util.List;
+import java.util.Optional;
+import org.apache.avro.Schema;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.hamcrest.CoreMatchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FakeKafkaServiceTest {
+
+  @Mock
+  private Schema avroSchema;
+  @Mock
+  private ProducerRecord producerRecord;
+  @Mock
+  private Record record;
+  private FakeKafkaRecord fakeKafkaRecord;
+
+  @Before
+  public void setUp() {
+    fakeKafkaRecord = FakeKafkaRecord.of(record, producerRecord);
+  }
+
+
+  @Test
+  public void shouldCreateTopicCorrectly() {
+    // Given:
+    final FakeKafkaService fakeKafkaService = FakeKafkaService.create();
```

Comment:
nit: can we pull these two lines into the `setUp()` method? They are shared by all three tests.